### PR TITLE
added Home page and some products

### PR DESCRIPTION
**Pull request by atharvabhanage02**
### Home Page

- Added Home Page with Featured Categories
- Redirects to the category clicked from Home Page to Product Listing Page
- Also added few more products
- Added button for go to Product Page if Cart/WishList is empty

Here is the Preview Link : https://deploy-preview-6--dainty-parfait-73cd3e.netlify.app/

### CODE REVIEW

File: src/Context/Product-Context/product-context.jsx [hunk 1]
```diff
@@ -44,6 +46,17 @@ const ProductProvider = ({ children }) => {
     })();
   }, []);
 
+    const [searchParams] = useSearchParams();
+    const categorygot = searchParams.get("categories");
```
Comment (Namrata-J) [inline src/Context/Product-Context/product-context.jsx:50]: suggestion : you could use better naming here, for an example say, selectedCategory.

File: src/Pages/Home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      <div>
+        <div
+          className="home-bg"
+          style={{ backgroundImage: "url('Assets/ecom-bg.jpg')" }}
+        >
+          <div className="home-text-section">
+            <h1 className="home-heading">Shop with Quality and Trust</h1>
+            <button className="shop-btn">
+              <Link className="shop-link-btn" to="/productlisting">
+                Shop Now{" "}
```
Comment (Namrata-J) [inline src/Pages/Home/Home.jsx:19]: is there any specific reason for having these empty curly braces, bcz if not you can remove them!!

**Review by Namrata-J — APPROVED** [review]

File: src/Pages/Home/Home.jsx (hunk 2 above)
Comment (atharvabhanage02) [reply src/Pages/Home/Home.jsx:19]: No , thats done by the Code Formatter , I'll remove that

File: src/Components/Cart-Card/Cart-Card.jsx [hunk 3]
```diff
@@ -20,7 +21,14 @@ export const CartCard = () => {
     });
   return (
     <div class="all-cart-products">
-      {cart.length === 0 ? <h1>Please add Items in Cart</h1> : null}
+      {cart.length === 0 ? (
```
Comment (munjalSaksham71) [inline src/Components/Cart-Card/Cart-Card.jsx:24]: If in else condition you are passing null, Better to use
```!cart.length && ()```

File: src/Components/Wishlist-Card/wishlist-card.jsx [hunk 4]
```diff
@@ -18,7 +19,14 @@ export const WishlistCard = () => {
 
   return (
     <div className="wishlist-wrapper">
-      {wishList.length === 0 ? <h1>Please add Items in WishList</h1> : null}
+      {wishList.length === 0 ? (
```
Comment (munjalSaksham71) [inline src/Components/Wishlist-Card/wishlist-card.jsx:22]: Same Here

**Review by munjalSaksham71 — APPROVED** [review]
Everything looks good! just give a one suggestion.

File: src/Components/Cart-Card/Cart-Card.jsx (hunk 3 above)
Comment (atharvabhanage02) [reply src/Components/Cart-Card/Cart-Card.jsx:24]: Yes , thats a better solution , thanks for suggesting that

**Pull request merged**